fe/fe-core/src/main/java/org/apache/doris/planner/external/iceberg/IcebergMetadataCache.java

```diff
@@ -201,15 +201,20 @@ private Table createIcebergTable(String uri, Map<String, String> hdfsConf, Strin
         HiveCatalog hiveCatalog = new HiveCatalog();
         hiveCatalog.setConf(conf);
 
-        Map<String, String> catalogProperties = new HashMap<>();
-        catalogProperties.put(HMSProperties.HIVE_METASTORE_URIS, uri);
-        catalogProperties.put("uri", uri);
-        hiveCatalog.initialize("hive", catalogProperties);
-
+        if (props.containsKey(HMSExternalCatalog.BIND_BROKER_NAME)) {
+            // Set Iceberg FileIO implementation as `IcebergBrokerIO` when Catalog binding broker is specified.
+            props.put("io-impl", "org.apache.doris.datasource.iceberg.broker.IcebergBrokerIO");
```

Comment:
I think this property should be added when creating catalog?

Reply:
OK, I'll edit the docs to illustrate the usage of this property